### ISSUE TITLE
Consistent error when searching kanji 二

### ISSUE THREAD

**Issue by GoogleCodeExporter** [nelenkov/wwwjdic#70]
```
What steps will reproduce the problem?
1. Change to Kanji search
2. Input Kanji "二" and search


What is the expected output? What do you see instead?
Expect to see the kanji page for "二" instead get error message.
I tested all other numbers and they work fine on their own.  Have tried 
switching to all available mirrors.  This is the only kanji I've found which is 
unsearchable.

What version of the product are you using? On what operating system?
version 2.2.5 on android 4.1.2

If possible, provide a stack trace (logcat).

As I can only test on HTC Magic and Nexus One, I can't reproduce device
specific issues.  Without a stack trace, there is usually nothing I can do.
So do attach logcat output. That or, send me a phone :)

Please provide any additional information below.

Don't think this is device specific, so I skipped this step.
```

Original issue reported on code.google.com by `PhillipL...@gmail.com` on 24 Feb 2013 at 4:38

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. You are right, it's not device specific. The entry for 
"二" has one extra space the parser doesn't expect. Fixed now, will be in the 
next release. 

I should really update the issue template :)

```

Original comment by `nikolay....@gmail.com` on 25 Feb 2013 at 8:23

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.3.
```

Original comment by `nikolay....@gmail.com` on 3 Jun 2013 at 5:08
- Changed state: **Fixed**
